### DISCUSSION TITLE
GSoC: Simulating Advanced Molecular Structures

### DISCUSSION THREAD

**Discussion by gluonparticle** [MDAnalysis/mdanalysis#4522] [GSoC Discussions]
Hi,. I am Sargun Singh. I am really excited about the opportunity to make a contribution to the sector of open-source Quantum Chemistry by working on the mission: "**Computational Framework for Simulating and Visualizing Complex Molecular Structures**". 

This is based on slight changes and some extra features from [this project](https://github.com/MDAnalysis/mdanalysis/wiki/GSoC-2024-Project-Ideas#project-4-2d-visualization-for-small-molecules) in the ideas list.

To assure you that I am the right person to contribute to this mission, I am really passionate about Science and am very experienced in the field of Chemistry, Physics, Mathematics, Statistics, Quantum Dynamics, Programming, and Machine Learning. 

I have worked on a similar project in high school where I targeted on simulating 4f orbitals in 3D using probability density, especially concentrated on the Cerium atom. This project concerned imposing algorithms to **calculate orbital possibilities, visualizing the effects in 3D space, and studying the traits of the orbitals** in Python. I have additionally shared this project recently on [this GitHub repository](https://github.com/gluonparticle/3D-Orbitals), and snapshots showcasing the simulations. I believe that my preceding experience aligns well with the goal of this mission. 

A detailed description of my project would include visualization of the detailed structure of atoms and molecules. I will add the ability to distinguish between different parts of atoms and molecules and their orbitals. Apart from the features already present in my previous project, the new features will include the addition of:

**- Showcasing electron clouds**
**- Probability of finding electrons based on bonds**  
**- Visualisation of complete molecules**
**- Developing a framework for visualising the reactions between atoms in real-time**
**- Consideration of other forces between two particles.**
**- Having a toggle option for viewing by orbitals or other models of atom**

I understand that I'm reaching out a bit later for the pre-proposal, but I wanted to take this opportunity to express my enthusiasm for collaborating with the mentors and MDAnalysis. Despite joining at this stage, I am highly motivated to contribute to the project and really eager to start working.

In addition to my technical abilities, I am a dedicated and enthusiastic learner who's keen to collaborate with mentors and fellow contributors to attain mission objectives. I am devoted to producing wonderful code, adhering to excellent practices, and actively collaborating inside the open-supply network.

If you have any queries, suggestions for additional features, or would like me to modify my proposal in any way, please feel free to let me know. I'm open to feedback and eager to ensure that my contributions align with the goals and expectations of MDAnalysis. 

Thank you for considering my software.

The Github repository link: https://github.com/gluonparticle/3D-Orbitals ;

**Comment by RMeli** [comment]
Thank you for participating in GSOC 2024 with us. Whether you were selected as a GSoC contributor or not, you are a valued member of the MDAnalysis community and we are excited to work with you outside of the GSoC program as well!
